xml/mysql_banners.xml

```diff
@@ -1071,7 +1071,7 @@
     <param pos="0" name="service.product" value="MySQL"/>
   </fingerprint>
 
-  <fingerprint pattern="^(\d{1,2}\.\d{1,3}\.\h{1,3})(?:[^+]+)?\+tld\d">
+  <fingerprint pattern="^(\d{1,2}\.\d{1,3}\.\h{1,3})([^+]+)?\+tld\d">
```

Comment:
Ah ok, I was worried about the captures, but didn't quite know how the change would affect them.  I'll keep playing with different ways of doing it and make sure to run the unit tests as well as the verify script.  I want to try and find a way of determining what the resulting regexp is after the replacement so we can just use that and the warning will go away.